script.js

```diff
@@ -41,6 +48,26 @@ function titleCaseInsensitive(showA, showB) {
   return 0;
 }
 
+function selectShowsMenu(e) {
+  let searchField = document.querySelector(".search");
+  searchField.value = "";
+  const wrapper = document.querySelector("#wrapper");
+  wrapper.innerHTML = "";
+  const countEpisodes = document.querySelector(".search-result");
+  const dropdownEpisodeMenu = document.querySelector(".episode-select");
+  dropdownEpisodeMenu.innerHTML = `<option class="episode-option" value="">All Episodes</option>`
+  const showId = e.currentTarget.value;
+  if (showId) {
+    const selectedUrl = `https://api.tvmaze.com/shows/${showId}/episodes`
```

Comment:
very nice and clean code askin well done. i am not sure why you fetch url two time but I am impressed with code code excellent.

Reply:
Hi Ali, thank you very much for the feedback that is invaluable. I only fetch the data with URL which the value either is passed from dropdown menu or header link.